cli/azd/cmd/infra_synth.go

```diff
@@ -24,22 +24,18 @@ import (
 
 type infraSynthFlags struct {
 	global *internal.GlobalCommandOptions
-	*internal.EnvFlag
-	force bool
+	force  bool
 }
 
 func newInfraSynthFlags(cmd *cobra.Command, global *internal.GlobalCommandOptions) *infraSynthFlags {
-	flags := &infraSynthFlags{
-		EnvFlag: &internal.EnvFlag{},
-	}
```

Comment:
`-e / --environment` flag is not actually supported or used for `azd infra synth`

Reply:
Infra synth actually does depend on the execution environment, but this relationship is buried deep in the bowels of our environment DI registration model [here](https://github.com/Azure/azure-dev/blob/b055b18b9da22a33b327dfe0f34d39637459bbbc/cli/azd/pkg/project/dotnet_importer.go#L392).

Reply:
Interesting - I will roll this back.